### PR TITLE
Add ZKPay Quest

### PR DESCRIPTION
**Pull request by zk-pay**
Changed text and added curatedAttributes to Aztec badge.
Created ZKPay quest.

### CODE REVIEW

**Comment by zk-pay** [comment]
https://etherscan.io/tx/0xcfba5bdb0866250db38772d1330748880b38450daf1962ffcd098e72279a714a

File: badges-metadata/main/hydra-s1-accountbound.ts [hunk 1]
```diff
@@ -885,6 +885,11 @@ export const hydraS1AccountboundBadges: BadgesCollection = {
           contact: "zk-pay",
         },
       ],
+      curatedAttributes: {
```
Comment (leosayous21) [inline badges-metadata/main/hydra-s1-accountbound.ts:888]: The curatedAttributes are added after the badge curation process https://www.notion.so/sismo/ZK-Badge-Curation-Guidelines-a62b5d7205734b44b41c603bf1516f4c
Can you remove them for now?

File: badges-metadata/main/hydra-s1-accountbound.ts [hunk 2]
```diff
@@ -906,6 +911,45 @@ export const hydraS1AccountboundBadges: BadgesCollection = {
...
+      description: "ZK Badge rewarded for ZKPay Quest completion",
+      image: "zkpay-quest.svg",
+      groupGeneratorName: "zkpay-quest",
+      publicContacts: [
+        {
+          type: "twitter",
+          contact: "@ZK_PAY",
+        },
+      ],
+      curatedAttributes: {
```
Comment (leosayous21) [inline badges-metadata/main/hydra-s1-accountbound.ts:927]: Same here for the curatedAttributes

**Review by leosayous21 — APPROVED** [review]

**Comment by leosayous21** [comment]
Your badge is now deployed, you will find it on https://app.sismo.io/zkpay-quest :-) 
Thanks!

**Pull request merged**